src/Tooling/Bit.Tooling.Templates/TodoTemplate/Web/Components/RedirectToSignIn.razor

```diff
@@ -2,9 +2,7 @@
 
 @code {
 
-    [Inject]
-    public NavigationManager NavigationManager { get; set; } = default!;
-
+    [Inject] public NavigationManager NavigationManager { get; set; } = default!;
```

Comment:
plz use the `@inject` directive instead of this inline injection and move it to the top of the file.